build.gradle.kts

```diff
@@ -58,3 +62,18 @@ kotlin {
 tasks.withType<Test> {
     useJUnitPlatform()
 }
+
+configure<DeltaCoverageConfiguration> {
+    val targetBranch = project.properties["diffBase"]?.toString() ?: "refs/remotes/origin/main"
+    diffSource.byGit {
+        compareWith(targetBranch)
+    }
+
+    violationRules.failIfCoverageLessThan(0.6)
+    reports {
+        html.set(true)
+    }
+}
+tasks.check {
```

Comment:
remove this dependsOn because you run deltacoverage twice

Reply:
Done